manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -302,25 +303,23 @@ public List<Processor> findByShardIdWithReadyDependencies(String shardId) {
 
     @Transactional
     @Override
-    public Processor updateProcessorStatus(ProcessorDTO processorDTO) {
-        Bridge bridge = bridgesService.getBridge(processorDTO.getBridgeId());
-        Processor p = processorDAO.findById(processorDTO.getId());
-        if (p == null) {
-            throw new ItemNotFoundException(String.format("Processor with id '%s' does not exist for Bridge '%s' for customer '%s'", bridge.getId(), bridge.getCustomerId(),
-                    processorDTO.getCustomerId()));
+    public Processor updateProcessorStatus(UpdateManagedResourceStatusDTO updateDTO) {
+        Processor p = processorDAO.findById(updateDTO.getId());
+        if (p == null || !p.getBridge().getCustomerId().equals(updateDTO.getCustomerId())) {
+            throw new ItemNotFoundException(String.format("Processor with id '%s' does not exist.", updateDTO.getId()));
```

Comment:
For example here it should be useful for troubleshooting to log BridgeID too

Reply:
As I've included the bridgeId in the updateDTO I've reverted this to the original implementation with the log statement